libr/bin/format/coff/coff.c

```diff
@@ -47,14 +50,54 @@ static bool r_coff_supported_arch_le(const ut8 *buf) {
 		return false;
 	}
 }
+#endif
 
+#ifndef R_BIN_COFF_BIGOBJ
 R_IPI bool r_coff_supported_arch(const ut8 *buf) {
 	return r_coff_supported_arch_le (buf) || r_coff_supported_arch_be (buf);
 }
+#endif
+
+// copied from bfd
+static bool r_coff_decode_base64(const char *str, ut32 len, ut32 *res) {
```

Comment:
Some time ago I had tested with the r2 functions but couldn't get it to work. According to this [thread](https://sourceware.org/bugzilla/show_bug.cgi?id=30444), they omit the padding characters. But I'll give it another shot.

Reply:
its ok for now, but if there's a bug in the r2 base64 apis it will be better to fix it there instead of having another duplicated implementation. thanks for taking care of this. i think the pr can be merged now! good work!